docs/reference/module-options.md

```diff
@@ -41,13 +41,42 @@ require('apostrophe')({
 });
 ```
 
+## Using module options
+
+Most module options in Apostrophe core and official extension modules are used automatically for specific purposes. No additional work is needed to use them for their original purposes other than configuring them, if needed. In some cases it is helpful to reference the options object in custom module code. You can do this in any of the module configuration function sections using the `self` argument as `self.options`.
```

Comment:
" No additional work is needed to use them for their original purposes other than configuring them, if needed." The 'if needed' feels weird here. No work is needed unless needed kind of vibe.

"In some cases it is helpful to reference the options object in custom module code." Are you saying its useful to reference our patterns when developing a custom module?

Reply:
How about just:

"Your custom modules can access options of their own via `self.options`."

Reply:
@localghost443 Sometimes you want to reference the options object in custom module code. Your use of "patterns" is throwing me off.

I wish we had a better term for the "module configuration function sections."

Reply:
Updating to:

> Module options can also be referenced directly in custom module code. Module configuration function sections take a `self` argument, which is the module itself. You can then get the options as `self.options`.